lib/fetch/util.js

```diff
@@ -297,6 +297,18 @@ function tryUpgradeRequestToAPotentiallyTrustworthyURL (request) {
   // TODO
 }
 
+// https://fetch.spec.whatwg.org/#body-fully-read
+function fullyReadBody (
+  // given an algorithm processBody
+  processBody,
+  // an algorithm processBodyError,
+  processBodyError,
+  // and an optional null, parallel queue, or global object taskDestination (default null)
+  taskDestination = null
+) {
+  // TODO
```

Comment:
You need to either consume the response or terminate fetch here or we will have stuck connections.

Reply:
This should be handled now in 64c9984b43dd662eb405da9f21d2e3271a7d4192 although I had to deviate from the spec slightly